cartridges/int_imgix_pd/cartridge/experience/components/imgix/imageComponent.json

```
@@ -1,5 +1,5 @@
 {
-  "name": "Image & Tittle",
+  "name": "PLEASE",
```

😆  maybe let's roll this back or change it to something like
```suggestion
  "name": "imgix Image",
```